### ISSUE TITLE
[OIII] line fitting errors

### ISSUE THREAD

**Issue by myblackhole01** [legolason/PyQSOFit#49]
Probably this a small issue from someone new to using PyQSOFit

I am trying to fit the [OIII] line using this code and would like to print out the errors of the fwhm, sigma, ew, peak, area for the  broad and narrow components.  Using these lines only prints out the Hbeta, I cant quite figure outwhich line to modify so that I can get these values for narrow and broad [OIII]. Any help will be appreciated .

print(q_mcmc.fur_result_name)
print('')
print(q_mcmc.fur_result)
print(np.shape(q_mcmc.fur_result))

['H$\\beta$_whole_br_fwhm' 'H$\\beta$_whole_br_fwhm_err'
 'H$\\beta$_whole_br_sigma' 'H$\\beta$_whole_br_sigma_err'
 'H$\\beta$_whole_br_ew' 'H$\\beta$_whole_br_ew_err'
 'H$\\beta$_whole_br_peak' 'H$\\beta$_whole_br_peak_err'
 'H$\\beta$_whole_br_area' 'H$\\beta$_whole_br_area_err']

[9.28420572e+03 2.48630566e-02 4.88994860e+03 8.95893372e-03
 5.51778618e+01 3.97228007e-05 4.85781746e+03 1.41261610e-04
 2.06497552e+03 4.46252091e-03]
(10,)

**Comment by burke86** [comment]
@legolason Is this because @myblackhole01 's OIII line does not actually satisfy the `ln_sigma_br > 0.0017` definition to be a "broad" line in the `line_prop` function?

Try computing them yourself with the `line_prop` function but lower the sigma_br threshold, I don't think there's a way to change this in the input to pyqsofit without changing the source file.

**Comment by myblackhole01** [comment]
So this function  q_mcmc.fur_result_name or q_mcmc.fur_result only outputs line fit results for broad lines? I would actually want to produce errors for both broad line and narrow line.

**Comment by burke86** [comment]
Yes, but you can compute what you are looking for yourself using `line_prop`  or `line_prop_prop_from_name` and gauss_result_all.

https://github.com/legolason/PyQSOFit/blob/a57f161a596bfed9ca24b9a6a4600564fde07026/pyqsofit/PyQSOFit.py#L1552

**Comment by legolason** [comment]
Yes, in our code, we will output all the parameters of the line > 1200 km/s, but for narrow lines, you need to calculate it by yourself, which is easy as burke86 pointed out, if you also want the error for OIII, you can set an save the line info externally and calculate the 16% and 84 percentile for errors.

**Comment by myblackhole01** [comment]
Thank you both.
Here is what I was trying to do to obtain errors of 50070III_w FWHM and sigma. I am just propagating the error from sigma to obtain that on FWHM. But that's not the issue, considering sigma alone, I get really very tiny errors. My understanding was that 
q_mc.line_result_name[48]
OIII5007w_1_sigma_err
gives a statistical error on sigma. From the results below, these errors are too small. Is there something I am missing?

fwhm, sigma, ew, peak, area,  = q_mc.line_prop(q_mc.linelist[5][0],q_mc.line_result[np.arange(43,48,2)],'broad')
print("OIII broad:")
print("FWHM (km/s)", np.round(fwhm,1), '+/-', np.round(fwhm,1)*q_mc.line_result[48].astype(np.float)*2.35)
print("Sigma (km/s)", np.round(sigma,1), '+/-', np.round(sigma,1)*q_mc.line_result[48].astype(np.float))

OIII broad:
FWHM (km/s) 1463.8 +/- 9.099230984945168e-07
Sigma (km/s) 612.5 +/- 1.6201722064922586e-07

**Comment by burke86** [comment]
@myblackhole01 Which version are you using? Try using the latest commit or the stable version (tag v1.1). There was a bug a few commits ago with the error calculation code, which I think is now fixed.
